### ISSUE TITLE
Update labels: Issue Template: project-profile-card-review-and-update

### ISSUE THREAD

**Issue by mjh-projects** [hackforla/website#4489]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.
### Overview
As developers, we need to add the labels from the [Missing series](https://github.com/hackforla/website/wiki/How-to-read-and-interpret-labels#missing-series) so that we can avoid the GitHub bot taking off the labels when devs add them to issues.
### Action Items
- [x] #4749
- [x] Replace the following line
   ```
   labels: ["P-Feature: Project Info and Page, role: back end/devOps, role: front end, Size: Small, size: 0.5pt, time sensitive"]
   ```
   with
   ```
   labels: ['P-Feature: Project Info and Page', 'role: back end/devOps', 'role: front end', ': Complexity: Small', 'size: 0.5pt', 'time sensitive']
   ```
- [x] Test the updated template using these [instructions](https://github.com/hackforla/website/issues/4307#issuecomment-1549888931).  Copy the contents of that comment into a new comment in this issue, and check off the boxes as you complete the steps.  

**For PR Reviewers and Merge Team**
To review this issue, click the link below under "Link for reviewers" and verify that the labels match the updated labels string in the issue.

**Link for Reviewers**
This URL will be inserted by the dev during the test process and will be used by reviewers to preview the updated template.
URL of the issue branch on the test Repository:  [REPLACE THIS TEXT AND BRACKETS WITH THE URL]  

**For PM, Merge Team, or Tech Lead**
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #4307 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `.github/ISSUE_TEMPLATE/project-profile-card-review-and-update.yml`
- This issue is part of epic #4307

**Comment by mlkara** [comment]
### Instructions for testing updated issue templates:
- [x] Push your issue branch to your fork of the website repository, as described in [Section 3.1.a of CONTRIBUTING doc](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#31a-push-all-changes-to-your-issue-branch), for example:  `git push --set-upstream origin test-update-labels-1234`  
- [x] The response from the push command will include the URL for creating a pull request.  Copy/Paste the URL below for use after testing:  
https://github.com/mlkara/website.git
- [x] Browse to your fork of the website repository and click Settings 
<details><summary>Screenshot of Repository Settings</summary>

![image](https://github.com/hackforla/website/assets/5314153/8ecf2e55-2187-4ab9-acf1-2fd7548291a1)

</details> 

- [x] Change the default branch to the new issue branch

<details><summary>Screenshot of Default Branch Setting</summary>

![image](https://github.com/hackforla/website/assets/5314153/945fe02e-035f-4215-9c25-1d46c209d8c0)

</details> 

- [x] Further down on the settings page under Features, check the Issues checkbox

<details><summary>Screenshot of Issues Checkbox</summary>

![image](https://github.com/hackforla/website/assets/5314153/945fe02e-035f-4215-9c25-1d46c209d8c0)

</details> 

- [x] Browse to the Issues page, then click Labels

<details><summary>Screenshot - creating labels in the test Repo</summary>

![image](https://github.com/hackforla/website/assets/5314153/208e1b07-2478-4ed3-98a1-207de58d25c5)

</details> 

- [x] Create labels corresponding to each label that was listed in the updated issue template.  The color of the labels don't have to match those on the `hackforla/website` repository, but the text of the labels must match exactly.  
- [x] From the Issues page, click `New Issue` then locate the template updated by your issue, and click `Get Started` to open the issue template. 
- [x] Copy/Paste the URL of the issue template into the Resources section of the issue, replacing the text: https://github.com/mlkara/website/issues/new?assignees=&labels=P-Feature%3A+Project+Info+and+Page%2Crole%3A+back+end%2FdevOps%2Crole%3A+front+end%2CComplexity%3A+Small%2Csize%3A+0.5pt%2Ctime+sensitive&projects=&template=project-profile-card-review-and-update.yml&title=Update+Project+Profile%3A+%5BProject+Name%5D
 This will enable others to check the template.  
- [x] Verify that the labels listed in the issue all appear on the issue template.  If there is a discrepancy between the labels listed in the issue and the labels that appear on the issue template, ask a dev lead or merge team member for help.  
- [x] If the labels listed in the issue match those listed on the issue template, complete the PR process by visiting the URL saved from the push command
- [x] Copy these sections from the Issue into the PR 
  - [x] "For PR Reviewers and Merge Team"
  - [x] "Link for Reviewers"
  - [x] "For PM, Merge Team, or Tech Lead"

**Comment by Sk-223** [comment]
### Instructions for testing updated issue templates:
- [x] Push your issue branch to your fork of the website repository, as described in [Section 3.1.a of CONTRIBUTING doc](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#31a-push-all-changes-to-your-issue-branch), for example:  `git push --set-upstream origin test-update-labels-1234`  
- [x] The response from the push command will include the URL for creating a pull request.  Copy/Paste the URL below for use after testing:  
[https://github.com/Sk-223/website]
- [x] Browse to your fork of the website repository and click Settings 
<details><summary>Screenshot of Repository Settings</summary>

![image](https://github.com/hackforla/website/assets/104045720/b22521f9-9bc5-4089-b69e-bef4d902ee81)


</details> 

- [x] Change the default branch to the new issue branch

<details><summary>Screenshot of Default Branch Setting</summary>

![image](https://github.com/hackforla/website/assets/104045720/0a7e7b08-920d-4552-9fba-10d857703ef0)
</details> 

- [x] Further down on the settings page under Features, check the Issues checkbox

<details><summary>Screenshot of Issues Checkbox</summary>

![image](https://github.com/hackforla/website/assets/104045720/219361a3-bd37-47d1-aac3-832bd16eb82a)

</details> 

- [x] Browse to the Issues page, then click Labels

<details><summary>Screenshot - creating labels in the test Repo</summary>

![image](https://github.com/hackforla/website/assets/5314153/208e1b07-2478-4ed3-98a1-207de58d25c5)

</details> 

- [x] Create labels corresponding to each label that was listed in the updated issue template.  The color of the labels don't have to match those on the `hackforla/website` repository, but the text of the labels must match exactly.  
- [x] From the Issues page, click `New Issue` then locate the template updated by your issue, and click `Get Started` to open the issue template. 
- [ ] Copy/Paste the URL of the issue template into the Resources section of the issue, replacing the text: [REPLACE THIS TEXT WITH URL OF THE ISSUE TEMPLATE ON THE TEST REPO]
 This will enable others to check the template.  
- [ ] Verify that the labels listed in the issue all appear on the issue template.  If there is a discrepancy between the labels listed in the issue and the labels that appear on the issue template, ask a dev lead or merge team member for help.  
- [ ] If the labels listed in the issue match those listed on the issue template, complete the PR process by visiting the URL saved from the push command
- [ ] Copy these sections from the Issue into the PR 
  - [ ] "For PR Reviewers and Merge Team"
  - [ ] "Link for Reviewers"
  - [ ] "For PM, Merge Team, or Tech Lead"

**Comment by t-will-gillis** [comment]
Hi @mlkara and @Sk-223 - you both assigned yourselves to this issue but this issue should be done by only one dev. 
- @ Sk-223: this issue was already picked up by @ mlkara when you assigned yourself, however
- @ mlkara:  when you assigned yourself to this issue, you did not move the issue from `Prioritized Backlog` into `In progress (actively working)` which is probably why @ Sk-223 thought this issue was available.

@Sk-223 unfortunately since this issue was already being worked on by @mlkara please un-assign yourself and find a new issue.

**Comment by roslynwythe** [comment]
Hi @mlkara -There was an error in the original issue, in the replacement text "labels" string.  I updated it, changing `Size: Small` to `Complexity: Small` and removing `Complexity: Missing`.  

After editing, adding and committing the `.yml` file, you'll have to push the branch again to your repository.  In the git response to the push, copy the URL provided for creating the Pull Request, and retain that URL.  In your repo you will have to create a new label "Complexity: Small".  Then bring up a new "Project Profile Card Review and Update" issue in your repo and copy the URL into the instructions, replacing the previous URL.  You can delete the square brackets.  After verifying that the labels match those in the updated issue, you will be ready to create the Pull Request. 

Thank you.  I apologize for the extra work caused by the error.
